### DISCUSSION TITLE
Easy duplicating QSO entry

### DISCUSSION THREAD

**Discussion by m0urs** [magicbug/Cloudlog#1869] [Ideas / Feature Requests]
Sometimes I am working a club station and the operator would like to have a QSL card for his personal call sign as well. So I would need an additional entry for his personal call sign besides the entry for the club station. Same is true for some POTA/WWFF/SOTA activation if there are more than one operator.

So I would like to have an easy way to duplicate an existing QSO e.g. by just pushing a button where all the data incl. date/time are already pre-filled with the original QSO and I just need to change the call.

This should be able to do in the live QSO list (by just clicking on a QSO in the list e.g.) but also later. E.g. sometime I just have logged the club station but later I get a QSL card via bureau also from the operator itself.

**Comment by magicbug** [comment]
There's already an open issue on this

**Reply by m0urs** [reply]
Sorry.  Did not find it but glad that it might be on the list!

**Reply by AndreasK79** [reply]
@magicbug are you sure? I can't locate that issue. This is a feature request for easy duplicating previous QSO then log with call entry.

**Reply by m0urs** [reply]
I did not find it either ;-) That was the reason why I opened anew one.

Happy Christmas to everyone!

**Reply by AndreasK79** [reply]
Thanks. Merry Christmas.
